dags/analyzer_helper/discourse/transform_raw_info.py

```diff
@@ -0,0 +1,93 @@
+from analyzer_helper.discourse.utils.convert_date_time_formats import DateTimeFormatConverter
+
+class TransformRawInfo:
+
+    def __init__(self):
+        self.converter = DateTimeFormatConverter()
+    
+    def create_data_entry(self, raw_data: list, interaction_type: str = None, interaction_user: int = None):
+        metadata = {
+            "channel_id": raw_data.get("category_id"),
+            "thread_id": raw_data.get("topic_id"),
+            "bot_activity": False,
+        }
+
+        if interaction_type == "reaction":
+            return {
```

Comment:
Please do not include multiple return statements in one method. This could be assigned to a variable and then returned after